docs/pvacseq/output_files.rst

```diff
@@ -172,80 +190,144 @@ all_epitopes.tsv and filtered.tsv Report Columns
+     - Tumor RNA depth at this position.
+   * - ``DNA_VAF``
+     - Tumor DNA variant allele frequency (VAF) at this position.
+   * - ``tier``
+     - A tier corresponding to the suitability of variants for use in vaccines.
 
-The pVACseq Neoeptiope Priority Rank
-____________________________________
+The pVACseq Aggregate Report Tiers
+__________________________________
 
-Each of the following 4 criteria are assigned a rank-ordered value (worst = 1):
+To bin a variant in a tier, the best binding epitope is evaluated as follows:
 
-- B = Rank of the mutant IC50 binding affinity, with the lowest being the best.
-  If the ``--top-score-metric`` is set to ``median`` (default) the ``Median MT Score``
-  is used. If it is set to ``lowest`` the ``Best MT Score`` is used.
-- F = Rank of ``Fold Change`` between MT and WT alleles, with the highest being the best.
-- M = Rank of mutant allele expression, calculated as (``Gene Expression`` * ``Tumor RNA VAF``), with the highest being the best.
-- D = Rank of ``Tumor DNA VAF``, with the highest being the best.
+.. list-table::
+   :header-rows: 1
 
-A score is calculated from the above ranks with the following formula: ``B + F + (M * 2) + (D / 2)``. This score is then converted to a rank (best = 1).
+   * - Tier
+     - Citeria
+   * - ``Pass``
+     - Median MT Score < 500 and (Tumor RNA VAF * Gene Expression) > 3 and
+       Tumor DNA VAF > (clonal VAF / 2) and not an anchor residue
+   * - ``Relaxed``
+     - Median MT Score < 1000 and (Tumor RNA VAF * Gene Expression) > 1 and
+       Tumor DNA VAF > (clonal VAF / 2) and not an anchor residue
+   * - ``Anchor``
+     - Median MT Score < 1000 and (Tumor RNA VAF * Gene Expression) > 1 and
+       Tumor DNA VAF > (clonal VAF / 2) and an anchor residue
+   * - ``Subclonal``
+     - Median MT Score < 1000 and (Tumor RNA VAF * Gene Expression) > 1 and
+       Tumor DNA VAF < (clonal VAF / 2) and not an anchor residue
+   * - ``LowExpr``
+     - Median MT Score < 1000 and low expression and
+       Tumor DNA VAF > (clonal VAF / 2) and not an anchor residue
+   * - ``NoExpr``
+     - Gene Expression == 0 and Tumor DNA VAF == 0 and expression is not low
+   * - ``Poor``
+     - Variant did not match any of the above criteria
 
-.. note::
+.. list-table::
+   :header-rows: 0
 
-   The pVACseq rank calculation detailed above is just one of many ways to prioritize neoeptiope candidates.
-   The body of evidence in this area is still incomplete, and the methodology of ranking is likely to change
-   substantially in future releases. While we have found this ranking useful, it is not a substitute for
-   careful curation and validation efforts.
+   * - anchor residue
+     - Anchor residue positions are definied as the first, second, second-to-last, and
+       last position in the epitope. The mutation in an epitope is an anchor residue if the mutation position is at one
+       of those positions and the median wildtype binding affinity is < 1000.
```

Comment:
Are we saying here that the definition of anchor residue in the tiers = 1) mutation at anchor residue + 2) median WT binding affinity is < 1000? If so, could we give it a different name to make it clearer? maybe something like "Anchor affected"?